site2/docs/client-libraries-go.md

```diff
@@ -302,6 +302,91 @@ fmt.Println(msg.Payload())
 canc()
 ```
 
+#### How to use Prometheus metrics in producer
+
+```go
+client, err := pulsar.NewClient(pulsar.ClientOptions{
+	URL: "pulsar://localhost:6650",
+})
+
+if err != nil {
+    log.Fatal(err)
+}
+
+defer client.Close()
+
+go func() {
+    prometheusPort := 2112
+    log.Printf("Starting Prometheus metrics at http://localhost:%v/metrics\n", prometheusPort)
+    http.Handle("/metrics", promhttp.Handler())
```

Comment:
How was `promhttp` created?

Reply:
I think `promhttp` is created by Prometheus module?
I found the demo at Prometheus official docs.
FYI: https://prometheus.io/docs/guides/go-application/#adding-your-own-metrics